package.json

```diff
@@ -30,21 +30,23 @@
       "raine"
     ]
   },
-  "main": "./lib",
+  "main": "./build/src",
   "scripts": {
     "build": "node scripts/build.js",
     "lint": "cross-env FORCE_COLOR=1 npm-run-all --parallel --aggregate-output lint:*",
```

Comment:
Unrelated but forgot to mention this the other day: why do we have `FORCE_COLOR=1` here? If it's for CI, this is now fixed there and locally it shouldn't be needed. It's one less devDependency :)

Reply:
It's not just for CI. `npm-run-all` appears to have some [known limitations with coloring](https://github.com/mysticatea/npm-run-all/blob/bf91f94ce597aa61da37d2e4208ce8c48bc86673/docs/npm-run-all.md#known-limitations). If I remove `FORCE_COLOR=1` and run it locally it loses color output.